misc/de-metas-common/de-metas-common-externalsystem/src/main/java/de/metas/common/externalsystem/JsonExternalSystemMessagePayload.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * #%L
+ * de-metas-common-externalsystem
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.common.externalsystem;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class JsonExternalSystemMessagePayload
+{
+	@JsonValue
+	String authToken;
+
+	public static JsonExternalSystemMessagePayload of(final String authToken)
+	{
+		return new JsonExternalSystemMessagePayload(authToken);
+	}
+
+	@JsonCreator
+	public JsonExternalSystemMessagePayload(
+			@NonNull final String authToken
+	)
+	{
+		this.authToken = authToken;
+	}
+}
```

Comment:
I think 

```
@JsonValue
	String authToken;
```

it's not such a great idea, wrapping the string with {"authToken":""} might help for clarity when checking messages

-> I suggest we go with a @Builder and @JsonDeserialize(builder)